src/main/resources/react4xp/_entries/Dashboard/Dashboard.jsx

```diff
@@ -153,7 +154,17 @@ class Dashboard extends React.Component {
     )
   }
 
+  renderAccordianForStatRegFetches () {
```

Comment:
**Note**: Trine's suggestion to move this to a separate section (instead of child in the existing accordion view)